sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -453,6 +501,13 @@ class PartialDependenceDisplay:
 
         .. versionadded:: 0.24
 
+    is_categorical : list of (bool,) or list of (bool, bool)
```

Comment:
Is possible to make a 2-way partial dependence plot where 1 of the features only is categorical, meaning something like: `is_categorical=[(True, False)]`?

Reply:
I believe not. That's what's being checked at https://github.com/scikit-learn/scikit-learn/pull/18298/files#diff-51e11eb022da44742c62c8a5d33c00d4R357-R359. Probably best to specify that in docs itself?

Reply:
So we could only accept `list of bool` and raise an error if `True` is given for a tuple.

Reply:
My idea was to follow the same shape of features (https://github.com/scikit-learn/scikit-learn/pull/18298/files#diff-51e11eb022da44742c62c8a5d33c00d4R453-R456) as discussed in https://github.com/scikit-learn/scikit-learn/issues/14969#issuecomment-679993674

Reply:
We can support `is_categorical=[(True, True)]` with a grid and colour of the cell denoting the partial dependency for the category combination. But, if this is not required, I am fine to go with a `list of bool` and raise an error if `True` is given for a tuple.

Reply:
Actually, we should replace it with `categorical_features` as well.
It might mean that we have to validate the parameter in the `Display` instead of the do it in the plot function.